shader/lighting/uber_lighting_helpers.glsl

```diff
@@ -97,9 +91,15 @@ layout (std430, binding = 8) buffer clusterBuffer
 
 struct clusterItem
 {
-    int lightIdx;
+    int lightIndex;
 };
 
+int lightIdxFromItem(in int item, in int part)
```

Comment:
unused?

Reply:
Removed. Was only in use when I checked the int32 to two int16 indices.